synapseAnnotations/data/common/minimal_Sage_standard.json

```diff
@@ -1045,6 +1055,41 @@
         "value": "CD8+ T-Cells",
         "description": "",
         "source": ""
+      },
+      {
+        "value": "CRISPR arachnoid",
```

Comment:
I think NF should supplement onc, and then when we apply annotations we just ignore all keys for which there are no values (see issue #112 )